Documentation/SwiftGenKit Contexts/files.md

```diff
@@ -0,0 +1,60 @@
+# Files parser
+
+## Input
+
+The files parser accepts multiple files or directories, which it will recursively search for files that match the given filter (default `.*`).
```

Comment:
Given that unlike any other parsers we have, this one is a bit special as it doesn't actually parse the _content_ of the files that it scans, but just their name and mimetype, I'd put a little more intro for this one, as it could be confusing for people who don't have more context to understand what it does ("but wait, what type of files does it parse? binary files? text files??")

For example maybe lead with:

> The files parser is intended to just list the name and mimetype of the files and subdirectories in a given directory. Unlike other parsers, this one doesn't parse the file _content_ per se, but only list those files so that you can point to them (get back their path or URL relative to your bundle) easily in your code.

Open for other wordings, I'm not super inspired and it's late here 😅 , but hopefully you see my point, the goal being to avoid the potential confusion for users that would think that this parser reads the content of the files like all our other parsers do.

Reply:
Thanks! Writing copy is a weak point of mine 😆